src/aat/java/uk/gov/hmcts/ccd/datastore/tests/AATHelper.java

```diff
@@ -58,4 +52,61 @@ public S2SHelper getS2SHelper() {
     public CCDHelper getCcdHelper() {
         return ccdHelper;
     }
+
+    public String getCaseworkerAutoTestEmail() {
+        return Env.require("CCD_CASEWORKER_AUTOTEST_EMAIL");
+    }
+
+    public String getCaseworkerAutoTestPassword() {
+        return Env.require("CCD_CASEWORKER_AUTOTEST_PASSWORD");
+    }
+
+    public String getDefinitionStoreUrl() {
+        return Env.require("CCD_DEFINITION_STORE_URL");
+    }
+
+    public String getImporterAutoTestEmail() {
+        return Env.require("CCD_IMPORT_AUTOTEST_EMAIL");
+    }
+
+    public String getImporterAutoTestPassword() {
+        return Env.require("CCD_IMPORT_AUTOTEST_PASSWORD");
+    }
+
+    public String getPrivateCaseworkerEmail() {
+        return Env.require("CCD_PRIVATE_CASEWORKER_EMAIL");
+    }
+
+    public String getPrivateCaseworkerPassword() {
+        return Env.require("CCD_PRIVATE_CASEWORKER_PASSWORD");
+    }
+
+    public String getRestrictedCaseworkerEmail() {
+        return Env.require("CCD_RESTRICTED_CASEWORKER_EMAIL");
+    }
+
+    public String getRestrictedCaseworkerPassword() {
+        return Env.require("CCD_RESTRICTED_CASEWORKER_PASSWORD");
+    }
+
+    public String getPrivateCaseworkerSolicitorEmail() {
+        return Env.require("CCD_PRIVATE_CASEWORKER_SOLICITOR_EMAIL");
+    }
+
+    public String getPrivateCaseworkerSolicitorPassword() {
+        return Env.require("CCD_PRIVATE_CASEWORKER_SOLICITOR_PASSWORD");
+    }
+
+    public String getElasticsearchBaseUri() {
+        return Env.require("ELASTIC_SEARCH_SCHEME") + "://" + Env.require("ELASTIC_SEARCH_HOST") + ":" + Env.require("ELASTIC_SEARCH_PORT");
+    }
+
+    public Long getLogstashReadDelay() {
```

Comment:
can we introduce a separate helper class for all the ES specific stuff like getLogstashReadDelay and getElasticsearchBaseUri etc..

Reply:
done